etc/create_brick.py

```diff
@@ -0,0 +1,36 @@
+import etcd
+import json
+import uuid
+
+
+integration_id = "ab3b125e-4769-4071-a349-e82b380c11f4"
+job_id = str(uuid.uuid4())
+
+payload = {
+    "integration_id": integration_id,
+    "run": "tendrl.gluster.flows.CreateGlusterBrick",
+    "parameters": {
+        "GlusterBrick.brick_dict": {
+            "12.34.56.78:/mnt/tendrl_sample4/b7": {
```

Comment:
Gdeploy currently has this limitation, We cant specify different configuration per brick. It has to be same for all the bricks passed.